.eslintrc.json

```diff
@@ -30,10 +30,26 @@
         "unicorn/no-array-reduce": "off",
         "unicorn/no-for-loop": "off",
         "unicorn/consistent-destructuring": "off",
-        "unicorn/prefer-switch": ["error", { "emptyDefaultCase": "do-nothing-comment" }],
-        "unicorn/number-literal-case": "off"
+        "unicorn/prefer-switch": ["error", { "emptyDefaultCase": "do-nothing-comment" }]
```

Comment:
arguably we could just drop this dependency at some point since we pull it in for one minor rule we probably don't really need